src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
SECURITY RISK: This method accepts `any` data and directly passes it to Prisma update. This could allow malicious users to update any user field, not just groups. This needs strict typing and validation.